pkg/checks/external/main.go

```diff
@@ -354,8 +354,8 @@ func (ext *Checker) setUUID(uuid string) error {
 			return fmt.Errorf("failed to verify uuid match %s after %d tries", checkState.Spec.CurrentUUID, tries)
 		}
 		tries++
-		ext.log("Waiting 1 second before checking " + ext.Name() + " uuid.")
-		time.Sleep(time.Second)
+		ext.log("Waiting 3 second before checking " + ext.Name() + " uuid.")
+		time.Sleep(time.Second * 3)
```

Comment:
I think we should up the limit of retries instead of upping the time in-between retries.

Reply:
Agree with @jonnydawg .  Could you change this on the `tries` variable instead please?  That way we don't add so much extra delay here...

Maybe set `tries` to a maximum of 9 and drop the sleep back down to 1s.

Reply:
done, changed from 3 secs to 1 and also up the number of tries to 9